### PR TITLE
release the build changes to use node 15 and npm 7

### PR DESCRIPTION
**Pull request by mario-subo**

### CODE REVIEW

File: .github/workflows/build-check.yml [hunk 1]
```diff
@@ -7,16 +7,16 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
       - name: Setup node
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
-      - name: Install npm@6
-        run: npm i npm@6 # to use workspace
+      - name: Install latest npm
+        run: npm i npm@7
```
Comment (kherP) [inline .github/workflows/build-check.yml:19]: actually we can remove this task

**Review by kherP — APPROVED** [review]

File: .github/workflows/build-check.yml (hunk 1 above)
Comment (mario-subo) [reply .github/workflows/build-check.yml:19]: oh yeah since it will install 7 anyway, will remove it

**Review by kherP — APPROVED** [review]

**Pull request merged**